.vs/VSWorkspaceState.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Do we need this file?

Reply:
Nope, sorry, wasn't thinking and ran the wrong add command

Reply:
Should be good now